src/shared/CMakeLists.txt.inc

```diff
@@ -12,10 +12,11 @@ include_directories(${shared_dir}/vartypes/enhanced)
 include_directories(${shared_dir}/vartypes/gui)
 include_directories(${shared_dir}/vartypes/xml)
 
-include_directories("/usr/include/eigen2")
+# include_directories("/usr/include/eigen2")  # update for eigen3 at toplevel 01/16
```

Comment:
Updated in change 89e70a6e2b6cebfd6f133d7e1e897600c2e8a85e (currently in testing)